packages/web-components/src/components/content-group-horizontal/__stories__/content-group-horizontal.stories.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import readme from './README.stories.mdx';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import '../../content-block/content-block-heading';
+import '../../content-block/content-block-complementary';
+import '../../content-item-horizontal/content-item-horizontal';
+import '../content-group-horizontal';
+
+const copy =
+  'Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. ' +
+  'Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. ' +
+  'Phasellus at elit sollicitudin.';
+
+const linkListItem = html`
+  <dds-content-item-horizontal .copy=${copy}>
```

Comment:
The markdown content for content pattern is now implemented as an inheritance of `<dds-markdown>`. Probably `<dds-content-item-copy>` for this case. Please refer to: #4351